docs/basic-usage/retrieving-media.md

```diff
@@ -20,6 +20,8 @@ $fullPathOnDisk = $mediaItems[0]->getPath();
 $temporaryS3Url = $mediaItems[0]->getTemporaryUrl(Carbon::now()->addMinutes(5));
 ```
 
+If you want to retrieve versioned media urls you can enable versioning by setting the `versioning` config to `true` in your `medialibrary.php` config file.
```

Comment:
Maybe add a few sentences on why you'd want to use this features (cache busting).